plugins/flashrom/flashrom.quirk

```diff
@@ -35,17 +39,33 @@ Plugin = flashrom
 [85aba599-addd-5985-a2e8-eddb41c61ba3]
 Plugin = flashrom
 
+# StarBook Mk VI - Intel (HwId)
+[5c917039-d938-5c9a-b22a-9c392b1534f3]
+Plugin = flashrom
+
+# StarBook Mk VI - AMD (HwId)
+[b39593ad-7522-52a5-a4ab-1b2ca2153956]
+Plugin = flashrom
+
+# StarBook Mk VIr2 - Intel (HwId)
+[6fca23e4-71e4-518a-96e2-b03ad7eb484f]
+Plugin = flashrom
+
+# Byte Mk I - AMD (HwId)
+[79a64046-5643-59c4-91d0-e68b33db5829]
+Plugin = flashrom
+
 # StarLabTop Mk III (coreboot GUID)
 [d33219e2-b84c-53a8-a624-27af9752dba6]
 Branch = coreboot
-VersionFormat = plain
```

Comment:
sure about this? I don't know if 1.7.x has a different default behaviour.

Reply:
Seems to work with or without it - will leave it on all to be safe